src/virt-install

```diff
@@ -174,6 +174,12 @@ for x in range(10):
 domain="coreos-inst-{}-{}".format(os.getpid(),int(time.time()))
 tail_proc = None
 http_proc = None
+
+# See the use of this below; also note virt-install also output the version to
+# stderr and not stdout.
+have_virtinstall_2 = int(subprocess.check_output(['virt-install', '--version'],
+                                                 stderr=subprocess.STDOUT).strip().decode('UTF-8').split('.')[0]) >= 2
```

Comment:
in #306 I needed to capture some output so I modified `run_verbose()` to be able to return a completed process so we could grab that output. I split out that commit into #310 so we could consider using it here.

Reply:
👍